misc/probe2trace.pl

```diff
@@ -64,6 +64,32 @@
     uint32_t latest;
 };
 
+struct quicly_cc_t {
+    uint32_t cwnd;
+    uint32_t ssthresh;
+    uint32_t stash;
+    uint64_t recovery_end;
+};
+
+struct quicly_stats_t {
+    uint8_t stats_version;
+    struct {
+        uint64_t received;
+        uint64_t decryption_failed;
+        uint64_t sent;
+        uint64_t lost;
+        uint64_t ack_received;
+        uint64_t late_acked;
+    } num_packets;
+    struct {
+        uint64_t received;
+        uint64_t sent;
+    } num_bytes;
+
+    struct quicly_rtt_t rtt;
+    struct quicly_cc_t cc;
+};
```

Comment:
This is fine, though I desparately hope that we can port this to h2olog soon, maybe as soon as [h2olog gets merged into h2o](https://github.com/h2o/h2o/pull/2353), so that we can avoid copy-pasting the struct here.